TrackletAlgorithm/MemoryTemplateBinnedCM.h

```diff
@@ -119,40 +133,42 @@ class MemoryTemplateBinnedCM{
 
   const DataType (&getMem(unsigned int icopy) const)[1<<NBIT_BX][1<<NBIT_ADDR] {
 #pragma HLS ARRAY_PARTITION variable=dataarray_ dim=1
+    if (isCMSSW) icopy = 0;
     return dataarray_[icopy];
   }
 
 
+#ifndef CMSSW_GIT_HASH
   const DataType (&get_mem() const)[NCOPY][1<<NBIT_BX][1<<NBIT_ADDR] {
     return dataarray_;
   }
+#endif
```

Comment:
@tomalin If `NCOPY` is replaced with `NCP`, does that allow you to remove the `#ifndef` protection?
```suggestion
  const DataType (&get_mem() const)[NCP][1<<NBIT_BX][1<<NBIT_ADDR] {
    return dataarray_;
  }
```

Reply:
The code would compile if I made your proposed change.. But this function would behave differently in CMSSW and non-CMSSW, which would potentially lead to different L1 tracking results inside and outside CMSSW. So I prefer to protect this function.